modules/terraform-zscc-gwlb-aws/variables.tf

```diff
@@ -105,3 +105,30 @@ variable "asg_enabled" {
   description = "Determines whether to set gwlb target group target_type to 'instance' or 'ip'. If set to true, ASG uses 'instance' and no aws_lb_target_group_attachment resources need to be created"
   default     = false
 }
+
+variable "flow_stickiness" {
+  type        = string
+  description = "Options are 5-tuple (src ip/src port/dest ip/dest port/protocol), 3-tuple (src ip/dest ip/protocol), or 2-tuple (src ip/dest ip). By default, Zscaler recommends 2-tuple to maintain flow stickiness to a specific target appliance. "
+  default     = "2-tuple"
```

Comment:
What is the reasoning behind changing from 5-tuple to 2-tuple?
Isn't default for both azure and aws 5-tuple?  Distribution of traffic is not uniform across all instances in the asg. 
 isn't it an issue in autoscale where some cc are loaded heavily and others not?  Is session stickiness important for the applications used by customers behind the CC and that is the reason for 2-tuple?

Reply:
azure default templates are 2-tuple. to be honest, most customers I would imagine would be fine with 5-tuple default and some very well change their azure deployments to 5-tuple. This was more of a @mksingh-zscaler and srikanth recommendation for corner case traffic like ftp so make sure it does not split out different DCs and/or NAT Gateways

Reply:
thanks